test/controllers/journeys/adjustments/profitOrLoss/ProfitOrLossCalculationControllerSpec.scala

```diff
@@ -174,5 +178,43 @@ class ProfitOrLossCalculationControllerSpec extends ControllerSpec with TableDri
         assertEqualWithDiff(contentAsString(result), expectedView)
       }
     }
+
+    s"should show message when income more then class 2 but less then class 4 threshold" in {
+      val taxableProfit: BigDecimal = 10000
+      val taxableProfitsAndLosses   = List(TaxableProfitAndLoss(taxableProfit = taxableProfit, taxableLoss = BigDecimal(0)))
+      val stubService = SelfEmploymentServiceStub(
+        getAllBusinessesTaxableProfitAndLossResult = Right(taxableProfitsAndLosses),
+        getUserDateOfBirthResult = Right(adultDoB)
```

Comment:
A valid DoB is the default response from the stub (already used in the 'should return OK and render correct view' of these tests) so can remove this line and `adultDoB `